client/app/scripts/superdesk-search/views/saved-searches.html

```diff
@@ -1,33 +1,36 @@
+<div class="search-filters">
+	<input id="search_saved_searches" ng-model="searchText" type="text" placeholder="{{:: 'Search saved searches' | translate}}">
+	<button id="search_saved_searches_button" ng-click="filter()">
+		<i class="svg-icon-right"></i>
+	</button>
+</div>   
 
-<div class="header">
-    <button class="btn btn-info" ng-click="edit()" translate>Save search</button>
-</div>
 <div class="list">
+	<div class="title" translate>Private Saved Searches</div>
+	<ul>
+	    <li ng-repeat="search in userSavedSearches | orderBy:'name' " ng-click="select(search)" ng-class="{active: search == selected}">
+	    	<div class="search-item">
+	    		<div class="search-name">{{:: search.name }} <i ng-if="search.is_global" translate>[Global]</i></div>
```

Comment:
May be one time binding isn't needed here as if user modifies name/description then that won't be reflected because of one time binding.